devtools/conda-envs/basic.yaml

```
@@ -49,5 +49,5 @@ dependencies:
   - h5py>=3.6.0
 
   # Optional
-  - openmmforcefields
+  - openmmforcefields ~=0.14.2
```

(not blocking) slight preference, otherwise this effectively becomes a downpin against 0.14.3. The patch I put into OMMFFs should make it compatible with both the old and new interchange behavior. 

```suggestion
  - openmmforcefields >=0.14.2
```

---

If I understand the standard[^1] correctly - and if conda follows `pip`'s behavior - this would allow 0.14.3 but _not_ 0.15.0. I read this as `>=0.14.2,<0.15.0`. My estimate is that 0.14.3 (through 0.14.9999999999....) would be less likely to include a behavior change than 0.15.0, same as a 1.0. I think this is what you're trying to get as well

[^1]: https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release

---

Ah, you're right. I mistakenly thought the `~=` operator was dependent on the precision of the version specified (so `~=0.14` would allow up to `0.14.999`, but `~=0.14.2` would only allow up to `0.14.2.999`). I now see that's not the case, so you can disregard this comment :-)